lib/compile.py

```diff
@@ -179,7 +183,9 @@ def compile_json(json_file):
         template = "default"
     script_contents = (script_templates[template] % script_desc)
 
-    new_script = open(json_file + '.py', 'w', encoding='utf-8')
-    new_script.write('# -*- coding: latin-1 -*-\n')
+    encoding = 'utf-8'
+    if encoding in json_object : encoding = json_object['encoding']
+    new_script = open(json_file + '.py', 'w', encoding=encoding)
+    new_script.write('# -*- coding: {} -*-\n'.format(encoding)) 
```

Comment:
This code is actually setting the encoding for the `.py` version of the `.json` file, not the encoding of the data, so I don't think we want changes here (unless I'm missing something).